### ISSUE TITLE
Errors During BUT HAVING NOTHING TO DO WITH Lag

### ISSUE THREAD

**Issue by R3P0FFXI** [LegionDark/Issues#1774]
**Date & Time**: 

1/7/2018 2:34PM EST

**FFXI Client Version  
(use `/ver` to see your version)**: 

**Server's Expected Client Version matches?  
(use `!ecv` to see expected version)  
(THIS IS A YES OR NO QUESTION!)**: 

**Character Name**: 

**Nation**: 

**Job(level)/Sub Job(level)**: 

**NPC or Monster or item Name**: 

**Zone name**: 

**Coordinates (use `!where`)**: 

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 

**Steps To Reproduce / any other info**: 

[07/Jan] [14:18:04][Error] luautils::onMobWeaponSkill[Skill ID: 1393]: scripts/globals/mob
skills/amatsu_tsukioboro_custom.lua:21: attempt to call method 'setSkillchain' (a nil valu
e)

[07/Jan] [14:20:33][Error] cmdhandler::call: (velvet): cannot open scripts/commands/velvet
.lua: No such file or directory

[07/Jan] [14:25:04][LUA Script] broken spell! ID: 26027 cast by: Hurkan params=3
[07/Jan] [14:25:04][Error] luautils::onSpellCast: .\scripts/globals/magic.lua:322: attempt
 to index local 'params' (a number value)

**Comment by TeoTwawki** [comment]
already known and working on

setskillchain is from custom crap in tenzens ws skills, I was middle of fixing them up proper 2 maints ago and it never got done because I kept being interrupted

"velvet" is just someone typing !velvet into their chat bar. seriously. look at there error. its an attempt at a non existing gm command.

hurkans using a bugged spell custom script. I started looking at it yesterday
